source/_patterns/01-atoms/breadcrumb/_breadcrumb.twig

```diff
@@ -14,12 +14,12 @@
 <nav aria-label="breadcrumb" role="navigation">
   <ol class="breadcrumb">
     {% set break = FALSE %}
-    {% for item in breadcrumb_items if not break %}
+    {% for item in breadcrumb if not break %}
```

Comment:
we can assume that the active breadcrumb will be the last breadcrumb in the list so this & the .yml can be cleaned up a bit with [twigs loop helpers](https://twig.symfony.com/doc/2.x/tags/for.html#the-loop-variable). both break and breadcrumb_active can be nixed.

`{% if loop.last %} active breadcrumb {% else %} normal breadcrumb {% endif %}`

will also need to add a small preprocess inside of patternlab.theme to fully populate the breadcrumb array (without it, I'm only seeing "Home" render as the active link, regardless of page)

```
/**
 * Implements hook_preprocess_breadcrumb().
 */
function patternlab_preprocess_breadcrumb(&$variables){
  if(($node = \Drupal::routeMatch()->getParameter('node')) && $variables['breadcrumb']){
    $variables['breadcrumb'][] = array(
      'text' => $node->getTitle()
    );
  }
}
```